### ISSUE TITLE
Where to change the tag focus indicator position?

### ISSUE THREAD

**Issue by linuxdotexe** [PapyElGringo/material-awesome#57]
The tag focus indicator position is to the left by defualt and I want it up or down since I moved the tag-list to the top panel but I cannot find it. Where can I do this customization?
